### ISSUE TITLE
A11y : Cannot close Threshold/Value-Mapping color-picker via keyboard 

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46490]
**Steps to reproduce**:

try to use the `set colour` button inside value mapping modal. use only a keyboard

**Actual Result**:

Button not usable via keyboard

**Expected Result**

workflow accessible for keyboard-only users

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
